Create the jenkins slave with python installed

jesschin opened the pull request:
This will be used by jenkins to spawn docker containers to test. Additionally, this also contains Docker Compose that will be used to build and deploy the SD4 instances for testing.

dlord commented:
We'll publish this on the ayannah public repository.

dlord commented on jenkins-slave/Dockerfile:
```diff
@@ -0,0 +1,43 @@
+    libsqlite3-dev \
+    wget \
+    curl \
+    llvm
```
Please run `rm -rf /var/lib/apt/lists/*` to remove any cached artifacts. This should make the image smaller.

dlord commented on jenkins-slave/id_rsa.pub:
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDHGwEPjsGSZeuELkUtvZrsaxKCzOmCE8zkzBEnRVCja5W2vwdHtjQR3ySkeUN+A9jHFaID8eIE7neQzs0v4TxkzIUvqx+v4l81grnlq5HcXx0LwVzqigZIAGbUGCEmJlrlXCqxXVRUP/aYO7Vb6RygM43gHt1hlWYaozsvdYRhf5gGBU3tnvTIxdSYt763W8ZyUFFcIA2PwCeQwn8rjqcQOwL+gNrPrqKQa729LkshHChTyW0mqaVTQtnJMaQ4+OmaadqCX/6iR90IdObniRhIdcAN5Sfve9EpBeALQtcy8gYunx997yHDH7wWJUu5PqleSWAcZBg9XUZ29zNXP5vL jchin@dev1.ayannah ...
```
Is it safe to actually include a static public key on our docker file?

jesschin committed:
Create the Dockerfile for building a jenkins slave

Includes a python installation (3.4.3), docker-compose, ssh keys and setup that will be used when jenkins will use
to use the slave. This was referenced from an existing code used to setup
chromatography. This also includes docker optimizations as suggested by
JP Alcala so that it will consume less space.

Signed-off-by: Jess Malvin Chin <jess@teamcodeflux.com>

dlord commented:
:+1:

Commit:
Added the Android SDK as part of the jenkins slave.

Signed-off-by: John Paul Alcala <jpalcala@ayannah.com>
